examples/example_segmentation_3d.ipynb

```diff
@@ -0,0 +1,482 @@
+    "For this project, the `SegmentationTrainer` is used and the class is initialized from the config file. \n",
+    "It is possible to override arguments in the config before passing it to the trainer. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Setuptools is replacing distutils.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2022-11-18 10:21:33,727 - No pushover credentials file submitted, will not try to push trainings progress to pushover device. If you want to receive status updated via pushover, provide the path to a yaml file, containing the `app_token`, `user_key` and `proxies` (optional) in the config at `pushover_credentials`\n"
+     ]
+    }
+   ],
+   "source": [
+    "trainer = SegmentationTrainer(config=config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With `show_batch`, `trainlib` provides a tool for quick visualization of the data. Data seen here, is passed directly to the model. \n",
+    "Because `ipywidgets` are used, the output is interactive and not visible once the notebook is shutdown. Masks can be toggled on/off and intensities can be changed. \n",
+    "```python\n",
+    "trainer.data_loader.show_batch()\n",
+    "```\n",
+    "![Show Batch Example](figures/example-show-batch.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`show_batch` uses `trainlib.viewer.ListViewer` to visualize output. But this class, as well as relatec classes can also be used directly in interactive sessions. \n",
+    "Monai re-orients images to comply with the NIfTI header. It might be therefore nessecary to re-arrange the array before viewing. Here, this is done by transposing the array. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from trainlib.viewer import BasicViewer, ListViewer, DicomExplorer\n",
+    "from monai.transforms import LoadImage\n",
+    "\n",
+    "reader = LoadImage(image_only=True)\n",
+    "image_1 = reader(\"../data/images/radiopaedia_10_85902_1.nii.gz\").transpose(0, 2)\n",
+    "image_2 = reader(\"../data/images/radiopaedia_10_85902_3.nii.gz\").transpose(0, 2)\n",
+    "\n",
+    "label_1 = reader(\"../data/labels/radiopaedia_10_85902_1.nii.gz\").transpose(0, 2)\n",
+    "label_2 = reader(\"../data/labels/radiopaedia_10_85902_3.nii.gz\").transpose(0, 2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`BasicViewer` can show a single image, with mask overlay or classification label. An optional description can also be added to the plot. \n",
+    "\n",
+    "```python\n",
+    "BasicViewer(image_1, label_1, description=\"A CT showing COVID Pneumonia\").show()\n",
```

Comment:
When click on `mask`, nothing happens. 

Maybe the mask should be something else than the image (is it the same currently)?